.github/workflows/build.yaml

```diff
@@ -20,19 +20,19 @@ jobs:
         run: |
           echo "IMAGE_TAG=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
       - name: Checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4.1.0
```

Comment:
I realize that this PR is already merged, but I'm going to ask anyway....

Changing this tag from "any `v3`" to "this particular patch of `v4`" isn't really much better than pinning it to a particular digest hash value (which we've already decided is a Bad Idea™).

@dustinblack, @jaredoconnell, shouldn't the tag be just `v4`?  (Analogously for the subsequent tags....)

Reply:
For Github actions, I'd go with just `v4`.